### ISSUE TITLE
Error messages not read out of context in GOV.UK Frontend review application legacy pages

### ISSUE THREAD

**Issue by nickcolley** [alphagov/govuk-frontend#1411]
This issue is from a May 2019 external accessibility audit report.

**WCAG Reference**: Usability feedback only, there is no WCAG related guidelines.
**Issue ID**: DAC_Issue27
**URL** http://govuk-frontend-v3.herokuapp.com/full-page-examples/applicant-details
**Page**: Applicants details

## Screen Shot
![](https://user-images.githubusercontent.com/2445413/58715668-47e9af00-83bf-11e9-91d1-8cc6cc967f9b.png)

The error message that appeared inline with the input field where an error had occurred, was not read out to screen reader users when navigating out of the context of the page as expected.

## Current Code Ref(s)
```html
<div class="form-group form-group-error "> <label class="form-label" for="full-name"> Full name <span class="form-hint"> Please enter your name as it’s written on official documents such as a passport or driving licence. </span> </label> <span class="error-message"> Enter your full name </span> <input class="form-control form-control-error " id="full-name" type="text" name="full-name" value=""> </div>
```

## Screen reader comments
> “I navigated in context when an error was committed and was able to locate the error message relating to the ‘First name’ field within the main content. However, when I viewed the edit field out of context, the error information did not announce relating to the field. This meant that the text indicated within the error skip link relating to the edit field could not be located using JAWS, NVDA and VoiceOver for iPhone. I would find it helpful if the error message could announce while navigating both in and out of context to ensure users can locate the error message.”

## Solution
Ensure that prefix of text ‘Error:’ is used to inform users that an error is present and associate the text with the field in question using aria-describedby.

**Comment by nickcolley** [comment]
This is an example of GOV.UK Elements code being run alongside GOV.UK Frontend to test for compatibility.

This failure is expected as a result of this.

We have improved this in GOV.UK Frontend, so as it's not a problem anymore I will close this issue.
